pkg/commands/refresh.go

```diff
@@ -52,8 +54,12 @@ func RefreshDefinitions(ctx context.Context, c client.Client, ioStreams cmdutil.
 	if err != nil {
 		return err
 	}
+	syncedTemplates = append(syncedTemplates, templates...)
 	ioStreams.Infof("get %d trait definitions from cluster, syncing...", len(templates))
 	successNum = plugins.SinkTemp2Local(templates, dir)
 	ioStreams.Infof("%d trait definitions successfully synced\n", successNum)
+
+	legacyNum := plugins.RemoveLegacyTemps(syncedTemplates, dir)
+	ioStreams.Infof("%d legacy capability definitions successfully removed\n", legacyNum)
```

Comment:
If `legacyNum` is equals to zero, the successful message will seem odd.

Reply:
yes, I will fix it.